### DISCUSSION TITLE
How can I get basketball match details with API?

### DISCUSSION THREAD

**Discussion by peterstewart0326** [cloudbet/docs#8] [Q&A]
I was trying to get basketball match score details using cloudbet API.
The API returned many data like markets and cutofftime, but not score details and current round time.
I found that in the api-response.md, there is "score" and "setScores" field in metadata, but I not at live online tester (https://docs.cloudbet.com/) and also with my code.
Is there any options or parameters to get score data (metadata)?

**Comment by kgravenreuth** [comment]
Hi @peterstewart0326 ,

We are currently not offering "score" and "setScores" fields currently and these fields have also been removed from https://github.com/Cloudbet/docs/blob/master/api-responses.md (in addition to the API docs and the actual API response itself).

The Cloudbet Feed API should primarily be used to obtain market information, as well as limited metadata information, such as the opinion of Cloudbet users on markets v/s the odds offered on the markets. You can also use the Trading API to place bets based on this market information.
